src/Symfony/Bundle/SecurityBundle/Tests/Functional/AccessTokenTest.php

```diff
@@ -425,11 +429,8 @@ public function testCasSuccess()
         $this->assertSame(['message' => 'Welcome @dunglas!'], json_decode($response->getContent(), true));
     }
 
-    public function validAccessTokens(): array
+    public static function validAccessTokens(): array
     {
-        if (!\extension_loaded('openssl')) {
```

Comment:
According to the Windows CI job running without `openssl`, this cannot be removed. The data provider is still executed before the check for `@requires extension openssl` skipping the tests.

Reply:
Oh right, I reverted this part